tutorials/migration.md

```diff
@@ -4,96 +4,846 @@ layout: intro
+Of particular node are the configuration folders and userdata folders. See the
+previous link for the location of these folders for your installtion.
+
+As with openHAB 1.x, one must restart openHAB 2 to pick up changes to .cfg files.
+Therefore restart openHAB 2 now.
+
+If you haven't already, configure default persistence now. You can do this 
+through `runtime.cfg` or PaperUI.
+
+Now openHAB 2 is ready to receive your other configurations, items and such. 
+Watch the log file as you transfer over files for errors. These errors will 
+have to be fixed.
+
+Copy over your files in the following order:
+
+~~~~
+cp <openHAB 1.x conf>/configurations/transform/* <openHAB 2 conf>/transform/*
+cp <openHAB 1.x conf>/configurations/scripts/* <openHAB 2 conf>/scripts/*
+cp <openHAB 1.x conf>/configurations/persistence/* <openHAB 2 conf>/persistence
+Copy any custom icons added from webapps/images to <openHAB 2 conf>/icons/classic
+Copy any custom webviews from webapps tp <openHAB 2 conf>/html
+cp <openHAB 1.x conf>/configurations/items/* <openHAB 2 conf>/items/*
+cp <openHAB 1.x conf>/configurations/rules/* <openHAB 2 conf>/rules/*
+cp <openHAB 1.x conf>/configurations/sitemaps/* <openHAB 2 conf>/sitemaps/*
+~~~~
+
+##Necessary Changes
+
+
+###Items
+
+As mentioned above, you must now manually install any Transformation engine your
+Items may use.
+
+The **SCALE** transformation has evolved.
+ 
+- Old syntax that was `[minbound,maxbound]` has to be changed to 
+`[minbound..maxbound]`. 
+- Note that you now have the ability to exclude bounds from the ranges (e.g. 
+`]minbound..maxbound]`) and also define open ranges`[minbound..]`
+
+
+###Sitemap
+
+If you use png icons, you must change the default icons from svg to png for
+ClassicUI and BasicUI. This can be done in PaperUI in `Configuration -> Service->BasicUI and ClassicUI`.
+Set the "Default Icon Format" to "Bitmap".
+
+Note that not all of the default icons that came with openHAB 1.x are avaialble 
+in the default set for openHAB 2. If you are missing an icon in your sitemap 
+that could be the cause. The fill list of openHAB 2 icons is [here](../addons/iconsets/classic/readme.html).
+
+Also, in openHAB 2 there is a new requirement for dynamic icons: there must be a 
+default. For example, if one has a bunch of Wunderground icons (e.g. 
+wunderground-chanceflurries.png) there mst be a `wunderground.png` icon as well.
+
+Note that the path to the Sitemap UIs have changed. If one goes
+to `http://<hostname>:8080` you will see a list of all the installed UIs, 
+including the administration UIs. To get to ClassicUI the URL is
+`http://<hostname>:8080/classicui/app?sitemap=<your sitemap>`. If you previously
+used `default.sitemap` as your file name so you could skip the `?sitemap=` part
+you must now rename the file to `_default.sitemap` and change the very first
+line of the file to read `sitemap _default label="<sitemap name>`. 
+
+openHAB 2 only supports static webviews. The dynamic webview some 
```

Comment:
I never was able to get it to work in OH 2 even with the compatibility layer. Perhaps the different path part was a problem. If you can provide some more details I'll make sure it gets added. appropriately.